### DISCUSSION TITLE
Release 0.2.0

### DISCUSSION THREAD

**Discussion by chalin** [google/docsy#965] [Announcement]
## Release summary

See, https://github.com/google/docsy/blob/main/CHANGELOG.md#020

## Commit details

* Turning docsy theme into a  hugo module by @deining in https://github.com/google/docsy/pull/801
* User guide: update outdated information by @deining in https://github.com/google/docsy/pull/812
* gh-827 Change height of rh sidebar to use all of screen by @at055612 in https://github.com/google/docsy/pull/828
* Fix for #801: enable imports for sites that are using git submodules by @deining in https://github.com/google/docsy/pull/837
* Page-meta-links: address .Path warning by @chalin in https://github.com/google/docsy/pull/842
* Changed blog/list.html to use .Title instead of .LinkTitle by @NaincyKumariKnoldus in https://github.com/google/docsy/pull/839
* Simplify setup: install Hugo automatically (and get the right version) using NPM by @chalin in https://github.com/google/docsy/pull/841
* Update es.toml by @Marzal in https://github.com/google/docsy/pull/815
* fix: there is no need existing two i18n/zh*.toml files by @h1z3y3 in https://github.com/google/docsy/pull/832
* CI: reintroduce explicit call to npm install by @chalin in https://github.com/google/docsy/pull/859
* Effectively revert #801, removing docsy as a Hugo module by @chalin in https://github.com/google/docsy/pull/860
* Plantuml svg loader by @erwin-faceit in https://github.com/google/docsy/pull/800
* i18n support: community page  by @h1z3y3 in https://github.com/google/docsy/pull/864
* Take `td-enable-google-fonts` into account for RTL fonts. by @SubOptimal in https://github.com/google/docsy/pull/869
* Pass correct arguments (TOCSS error) by @victoriadrake in https://github.com/google/docsy/pull/875
* feat: optimize i18n of community by @h1z3y3 in https://github.com/google/docsy/pull/868
* Revert "Pass correct arguments (TOCSS error) (#875)" by @chalin in https://github.com/google/docsy/pull/885
* Upgrade to Hugo 0.92.1 by @chalin in https://github.com/google/docsy/pull/886
* Revert back to Bootstrap v4.6.1 by @chalin in https://github.com/google/docsy/pull/887
* README: markdown cleanup, line-wrapping, etc by @chalin in https://github.com/google/docsy/pull/892
* NPM script refactoring and README cmd adjustment by @chalin in https://github.com/google/docsy/pull/893
* Upgrade to Hugo 0.92.2 and add NPM package name, etc by @chalin in https://github.com/google/docsy/pull/896
* Enable to pull in docsy as hugo module by @deining in https://github.com/google/docsy/pull/871
* Taxonomy bugfix by @raum51 in https://github.com/google/docsy/pull/895
* User guide config: use mounts rather than the old contentDir param by @chalin in https://github.com/google/docsy/pull/898
* Netlify: use docs-install before building the User Guide by @chalin in https://github.com/google/docsy/pull/902
* Add support for Bootstrap and FontAwesome via NPM packages by @chalin in https://github.com/google/docsy/pull/904
* Fix: shortcode 'lead' produces invalid HTML by @deining in https://github.com/google/docsy/pull/905
* Upgrade Hugo to 0.93.2 by @chalin in https://github.com/google/docsy/pull/911
* Update fr.toml by @arnaudlevy in https://github.com/google/docsy/pull/910
* Fix taxonomy cloud and article terms by @raum51 in https://github.com/google/docsy/pull/909
* User guide index cleanup by @chalin in https://github.com/google/docsy/pull/915
* Add Deploys-by-Netlify badge to UG homepage by @chalin in https://github.com/google/docsy/pull/932
* Reflect renaming of master branch (various places) by @deining in https://github.com/google/docsy/pull/916
* User guide: fix broken anchor links by @deining in https://github.com/google/docsy/pull/922
* Restore get:submodule script by @chalin in https://github.com/google/docsy/pull/938
* CI: don't clean assets/vendor for local builds by @chalin in https://github.com/google/docsy/pull/940
* User guide: curate main menu by @chalin in https://github.com/google/docsy/pull/945
* Add an empty blog by @chalin in https://github.com/google/docsy/pull/946
* Drop `shallow=true` config for git submodules by @chalin in https://github.com/google/docsy/pull/952
* User guide: adding instructions on how to use and update docsy as hugo module by @deining in https://github.com/google/docsy/pull/802
* Added Ukrainian translation, completed Russian translation by @deining in https://github.com/google/docsy/pull/959
* User guide, chapter 'Get started': fix broken link, add alias by @deining in https://github.com/google/docsy/pull/963
* Create CHANGELOG.md by @chalin in https://github.com/google/docsy/pull/962
* NPM package: change version to 0.2.0 by @chalin in https://github.com/google/docsy/pull/961
* Prepare docu and repo for release v0.2.0 by @deining in https://github.com/google/docsy/pull/958
* create first Docsy blog post by @emckean in https://github.com/google/docsy/pull/953
* README: add link to Netlify dashboard by @chalin in https://github.com/google/docsy/pull/964

**Full Changelog**: https://github.com/google/docsy/compare/v0.1.0...v0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/google/docsy/releases/tag/v0.2.0'>0.2.0</a>.</em>

**Comment by chalin** [comment]
Apologies to all who experienced the turbulence as we landed this release (our second attempt). Thanks to all who gave feedback (#954), we're expecting to offer a smoother (pre-) release experience moving forward.

**Reply by jcrummy** [reply]
All good. This is still my favorite theme for documentation, so I'm excited about some of the changes!

**Comment by okohll** [comment]
Hey, congratulations on the 0.2 release. I look forward to upgrading. One comment on the documentation, the link to the migration guide under

https://www.docsy.dev/docs/get-started/#migration-and-backward-compatibility

just seems to go the same page. Is there actually a migration guide?

**Reply by deining** [reply]
> Hey, congratulations on the 0.2 release. 

You are welcome.

>I look forward to upgrading.

Good luck with your upgrade!

> One comment on the documentation, the link to the migration guide under
> 
> https://www.docsy.dev/docs/get-started/#migration-and-backward-compatibility
> 
> just seems to go the same page. Is there actually a migration guide?

Yes there is a [migration guide](https://www.docsy.dev/docs/updating/convert-site-to-module/) available! 😄

Thanks for pointing the incorrect link out! I will author a fix/PR for that shortly,

**Reply by okohll** [reply]
Thanks, the upgrade to our site https://docs.agilebase.co.uk/ seems to have gone smoothly
